components/automate-backend-opensearch/habitat/config/jvm.options

```diff
@@ -19,8 +19,8 @@
 # Xms represents the initial size of total heap space
 # Xmx represents the maximum size of total heap space
 
--Xms1g
--Xmx1g
+-Xms{{cfg.runtime.heapsize}}
+-Xmx{{cfg.runtime.heapsize}}
```

Comment:
should these not be 2 different variables, xms is initial memory blocked on RAM
xmx is max memory allocated for JVM on RAM